controller/instance_manager_controller.go

```diff
@@ -1662,13 +1662,11 @@ func (imc *InstanceManagerController) createInstanceManagerPodSpec(im *longhorn.
 
 func (imc *InstanceManagerController) deleteOrphans(im *longhorn.InstanceManager, isInstanceManagerTerminating bool) error {
 	autoDeletionTypes, err := imc.ds.GetSettingOrphanResourceAutoDeletion()
-	var autoDeleteEngine = false
-	var autoDeleteReplica = false
+	var autoDelete = false
```

Comment:
Why not move `autoDelete` into `areOrphanAndResourceDeletable`?

Reply:
If there's some errors on the `orphan-resource-auto-deletion` setting, it will keep logging over the orphan iteration loop.